From 5c574dcc64ff64160329d93576df25f9e186e4e3 Mon Sep 17 00:00:00 2001
From: Natanael Copa <ncopa@alpinelinux.org>
Date: Wed, 8 May 2013 13:42:57 +0200
Subject: [PATCH] set terminfo directory in config.mk

This helps packagers to install the terminfo at proper place rather than
in users homedir.

Signed-off-by: Natanael Copa <ncopa@alpinelinux.org>
---
 Makefile  | 2 +-
 config.mk | 1 +
 2 files changed, 2 insertions(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 52af636..3ac74a1 100644
--- a/Makefile
+++ b/Makefile
@@ -49,7 +49,7 @@ install: all
 	@sed "s/VERSION/${VERSION}/g" < st.1 > ${DESTDIR}${MANPREFIX}/man1/st.1
 	@chmod 644 ${DESTDIR}${MANPREFIX}/man1/st.1
 	@echo Please see the README file regarding the terminfo entry of st.
-	@tic -s st.info
+	@tic -o ${DESTDIR}${TERMINFODIR} -s st.info
 
 uninstall:
 	@echo removing executable file from ${DESTDIR}${PREFIX}/bin
diff --git a/config.mk b/config.mk
index 9431de2..3ac9bac 100644
--- a/config.mk
+++ b/config.mk
@@ -6,6 +6,7 @@ VERSION = 0.4.1
 # paths
 PREFIX = /usr/local
 MANPREFIX = ${PREFIX}/share/man
+TERMINFODIR = $$HOME/.terminfo
 
 X11INC = /usr/X11R6/include
 X11LIB = /usr/X11R6/lib
-- 
1.8.2.2

